Terasology/GooKeeper
# fix(gltf): replace deprecated md5 assets

# How to test

1. Start a `GooKeeper` gameplay
2. Take care that the gooeys don't kill you :sweat_smile: 
3. Use `spawnPrefab <prefab>` command in the in-game terminal (open via `F1`) to spawn the following prefabs
   * blueGooey
   * yellowGooey
   * redGooey
   * gooeyHatchling
   * visitor (will likely crash the game without changes from https://github.com/Terasology/GooKeeper/pull/26 being present)
4. Check that the models are visible and the height offset seems about right (fully above the ground but not obviously floating in the air)
